src/aap_eda/settings/default.py

```diff
@@ -403,14 +414,24 @@ def _rq_redis_client_additional_parameters():
     return params
 
 
-def rq_redis_client_instantiation_parameters():
+def rq_standalone_redis_client_instantiation_parameters():
     params = _rq_common_parameters() | _rq_redis_client_additional_parameters()
 
     # Convert to lowercase for use in instantiating a redis client.
     params = {k.lower(): v for (k, v) in params.items()}
     return params
 
 
+def rq_redis_client_instantiation_parameters():
+    params = rq_standalone_redis_client_instantiation_parameters()
+
+    # Include the HA cluster parameters.
+    params["clustered"] = bool(REDIS_HA_CLUSTER_HOSTS)
+    params["clustered_hosts"] = REDIS_HA_CLUSTER_HOSTS
```

Comment:
Not sure if you need to update the DAB version included in EDA but the clustered/clustered_hosts options were changed not so long ago

```suggestion
    if bool(REDIS_HA_CLUSTER_HOSTS):
        params["mode"] = "cluster"
    params["redis_hosts"] = REDIS_HA_CLUSTER_HOSTS
```

https://github.com/ansible/django-ansible-base/commit/6ff14f82046cfa565f9fdcc05f9ec4c1a2011bb6

Reply:
@dsavineau Updated to latest.